doc/holochain_101/src/testing_configuration.md

```diff
@@ -112,36 +112,104 @@ console.log(dnaConfig)
 */
 ```
 
+## Bridges
+
+### `Config.bridge(handle, callerInstanceConfig, calleeInstanceConfig)` => `object`
+
+Takes three arguments: the bridge handle, the caller, and the callee (both instances)
+
+___
+
+**Name** handle
+
+**Type** `string`
+
+**Description** The desired bridge handle, as required by your DNA. See the [bridging section of the docs](./bridging.md) for more detail
+
+___
+
+**Name** callerInstanceConfig
+
+**Type** `object`
+
+**Description** A config object with a `name` property, as produced by `Config.instance`
```

Comment:
Can you state a very simple explanation of what the 'caller' is, in a bridging scenario?